### DISCUSSION TITLE
Why not using floppy instead of http ?

### DISCUSSION THREAD

**Discussion by metabsd** [vmware/packer-examples-for-vsphere#63] [Help] [answered]
I'm trying to modify my fork with this because of lack of dhcp.

**Comment by tenthirtyam** [comment, marked as answer]
Not all Linux images support floppy - for example, any RHEL8 variants.

`main` has an update to support `disk` which will use `cd_content` for the Linux images that support kickstart via cdrom. It be merged into the included release.

Ryan

**Reply by metabsd** [reply]
Can you please refer commit that I can refer to please 👍🏼

**Reply by tenthirtyam** [reply]
https://github.com/rainpole/packer-vsphere/pull/59

**Reply by metabsd** [reply]
Thx! I really appreciate it. Have a nice day!
